### DISCUSSION TITLE
SFX isolation

### DISCUSSION THREAD

**Discussion by PeradoxSRj** [Anjok07/ultimatevocalremovergui#1287] [Q&A]
Which model is best for sfx extraction?

**Comment by FlashlightET** [comment]
mvsep has Demucs4HT DNR but ~~unfortunately mvsep refuses to make the model public~~ i just wasnt looking hard enough lol

update: i found this actually and am about to test it https://github.com/ZFTurbo/MVSEP-CDX23-Cinematic-Sound-Demixing

update: it didnt really sound that good (in my thrown together attempt to load the model)

i can not get it to sound good. There is the bandit model though at https://github.com/karnwatcharasupat/bandit ~~but there isnt any guide out there on how to set it up, not even a requirements.txt.~~ I am so dumb there is an environment.yaml

**Reply by NLesha** [reply]
Can you show how to import it into UVR?

Thanks.

**Comment by GUUser91** [comment]
Music Source Separation Training by ZFTurbo has the bandit plus model which does a decent job at separating sound effects.
https://github.com/ZFTurbo/Music-Source-Separation-Training/blob/main/docs/pretrained_models.md
https://github.com/ZFTurbo/Music-Source-Separation-Training/blob/main/docs/gui.md
Sample:
Unaltered audio:
https://vocaroo.com/1b24SWu7CEsL
SFX from using bandit
https://vocaroo.com/1k9KJqX13nUW

mvsep has MVSep DnR v3 and Demucs4HT DNR
https://mvsep.com/en
MVSep DnR v3 Sample:
Unaltered audio:
https://vocaroo.com/1b24SWu7CEsL
SFX from using MVSep DnR v3
https://vocaroo.com/14fB6GGZrkSH

If you're willing to pay, you can purchase the moises pro plan. It has the best sound effect separator in my opinion. I purchased the yearly plan during a black friday sale for $120
Sample:
Unaltered audio:
https://vocaroo.com/1b24SWu7CEsL
SFX from using moises
https://vocaroo.com/1mJ7PEZ5qJ6q
